linetools/isgm/abscomponent.py

```diff
@@ -411,6 +422,32 @@ def stack_plot(self, nrow=6, show=True):
             plt.show()
         plt.close()
 
+
+    def repr_vpfit(self, b=10.):
```

Comment:
Please add in the docstring whether this is fort.26 or fort.13 format. I always get confused between the two.